app/components/PolicyContainer.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect, useState } from 'react';
+import logo from '../assets/full_logo.png';
+
+type ParsedLine = {
+  type:
+    | 'paragraph'
+    | 'heading1'
+    | 'heading2'
+    | 'heading3'
+    | 'listItem'
+    | 'break';
+  content: string;
+};
+
+type PolicyContainerProps = {
+  content: string;
+};
+
+export function PolicyContainer({ content }: PolicyContainerProps) {
+  const [parsedContent, setParsedContent] = useState<ParsedLine[]>([]);
+
+  // Simple markdown parser without external libraries
```

Comment:
is this something that you built yourself? are there some other possible markdown line types or these are all? if there are more we might need to maintain this stuff every time we use something new